docs/faq.md

```diff
@@ -22,10 +22,10 @@ import TOCInline from "@theme/TOCInline"
 
 #### Semgrep Community Edition (CE)
 
-Semgrep Community Edition is a free, community-supported code scanning tool. It's perfect for individuals, security auditors, and pentesters who need fast, one-off scans. You can use it at work, on private and proprietary code, no problem!
+Semgrep Community Edition is a free, community-supported code scanning tool. It's perfect for individuals, security auditors, and penetration testers who need fast, one-off scans. You can use it at work, on private and proprietary code, no problem!
 
 Semgrep Community Edition includes:
-* The Semgrep [open-source engine](https://github.com/semgrep/semgrep): Governed by the [LGPL 2.1](https://tldrlegal.com/license/gnu-lesser-general-public-license-v2.1-(lgpl-2.1)) license
+* The Semgrep [open source engine](https://github.com/semgrep/semgrep): Governed by the [LGPL 2.1](https://tldrlegal.com/license/gnu-lesser-general-public-license-v2.1-(lgpl-2.1)) license
 * The Semgrep [Community Rules](https://github.com/semgrep/semgrep-rules/): Governed by the [Semgrep Rules License v. 1.0](https://semgrep.dev/legal/rules-license/)
```

Comment:
```suggestion
* [Semgrep-maintained Rules](https://github.com/semgrep/semgrep-rules/): Governed by the [Semgrep Rules License v. 1.0](https://semgrep.dev/legal/rules-license/)
```

Reply:
> 

Doesn’t Semgrep also maintain rules that are not governed by that license (i.e., Pro rules)? That’s the reason I didn’t go with “Semgrep-maintained rules” initially.

Reply:
It links to `semgrep-rules`, which is all the rules we maintain, even if we didn't author (all) those rules; the **Semgrep Registry** is all the rules from `semgrep-rules`, + others. We don't have a name for non-Pro rules that we maintain, is the problem I'm seeing here. The most neutral suggestion is to display the bare link (`semgrep-rules` repository). I will update the comment.

Reply:
Another question is - can we consider the Semgrep Repository a part of "Semgrep CE"? (It has rules from Trail of Bits and other cybersecurity companies, though).